pulsar-client/src/main/java/org/apache/pulsar/client/impl/TableViewImpl.java

```diff
@@ -106,6 +106,7 @@ private void checkForPartitionsChanges(Timeout timeout) {
         start().whenComplete((tw, ex) -> {
            if (ex != null) {
                log.warn("Failed to check for changes in number of partitions");
```

Comment:
I think @eolivelli was meaning to include the exception in the log message here.